code/modules/mob/living/simple_mob/simple_mob.dm

```diff
@@ -59,8 +60,7 @@
 	var/harm_intent_damage = 3		// How much an unarmed harm click does to this mob.
 	var/list/loot_list = list()		// The list of lootable objects to drop, with "/path = prob%" structure
 	var/obj/item/weapon/card/id/myid// An ID card if they have one to give them access to stuff.
-	var/list/organ_names = list("body") //Bodyparts that can be shown as hit by projectiles.
-
+	var organ_names = /decl/mob_organ_names //'False' bodyparts that can be shown as hit by projectiles in place of the default humanoid bodyplan.
```

Comment:
Missing `/`

Reply:
Not sure how you managed to comment this in the like 5 minutes before I put up the last push that fixed this!

Reply:
Maint status is a pathway to abilities many would call... unnatural.